test/spec/modules/prebidServerBidAdapter_spec.js

```diff
@@ -2155,6 +2155,15 @@ describe('S2S Adapter', function () {
       })
     });
 
+    it('should set default s2s ttl', function () {
+      config.setConfig({
+        s2sConfig: {
+          defaultS2sTtl: 30
+        }
+      });
+      expect(config.getConfig('s2sConfig').defaultS2sTtl).to.deep.equal(30)
```

Comment:
This should look at bidObject.ttl - this test is only confirming the setConfig/getConfig, not the code above that defaults the bidObject.ttl

Reply:
makes sense

Reply:
i wrote two new tests, one extended the test of the current response to have ttl 60; the other changed it to 30 and checked that response. Thanks for the feedback!